doc/netplan-yaml.md

```diff
@@ -1144,7 +1144,7 @@ wpasupplicant installed if you let the `networkd` renderer handle wifi.
     > `bandwidth`, and `count`. This option is only used in 802.3ad
     > mode.
 
-  - **all-slaves-active** (bool)
+  - **all-members-active**, **all-slaves-active** (bool)
```

Comment:
I'd rather put the "all-slaves-active" title down into the description, e.g. as `Alias: **all-slaves-active**`